src/preact/index.js

```diff
@@ -22,6 +23,8 @@ export {
   useImperativeHandle,
 } from 'preact/hooks';
 
+export {forwardRef} from 'preact/compat';
```

Comment:
We actually exclude this because it causes a large bundle size increase. We have a helper at `#preact/compat` that can be DCE'd more efficiently. You'll have to update your `src/preact/alias.js` code to import from both and export a cohesive default object.